### DISCUSSION TITLE
Need help to get started

### DISCUSSION THREAD

**Discussion by rakeshVerse** [appwrite/appwrite#5108] [Q&A] [answered]
Hello guys, I'm a PHP full stack developer. I'm willing to contribute to AppWrite. I went through the code base but couldn't understand anything. 
Could you please guide me on how can I understand the project better and share some resources for Tech Stack used to fill skill gaps, if any?

**Comment by stnguyen90** [comment, marked as answer]
Maybe this will help: https://github.com/appwrite/appwrite/discussions/4899#discussioncomment-4478125
